Makefile

```
@@ -1,3 +1,6 @@
+PROTO_DIR = Tests/Protobuf
+OUT_DIR = Tests/CombineGRPCTests/Generated
```

Perhaps this, to be explicit?

```suggestion
PROTO_GEN_DIR = Tests/CombineGRPCTests/Generated
```